packages/mephisto-worker-experience/src/index.css

```diff
@@ -0,0 +1,26 @@
+.tips-list {
+  list-style-type: none;
+  padding: 0;
+}
+.tip {
```

Comment:
we should definitely make these class names more specific.

having something as simple as `.tip` can cause unintended conflicts with other libraries that the user might be pulling in, or with css that they've written themselves.

I propose prefixing the class names here with some namespacing to avoid such collisions. For example:

`.tip` -> `.mephisto-worker-experience-tips__tip`
`.submit-button` -> `.mephisto-worker-experience-tips__submit-button`